proprietary/tokens/src/components/amsterdam/alert.tokens.json

```diff
@@ -4,10 +4,10 @@
       "border-width": { "value": "4px" },
       "border-style": { "value": "solid" },
       "gap": { "value": "1rem" },
-      "padding-block-start": { "value": "1rem" },
-      "padding-block-end": { "value": "1rem" },
-      "padding-inline-start": { "value": "1.5rem" },
-      "padding-inline-end": { "value": "1.5rem" },
+      "padding-block-start": { "value": "{amsterdam.space.inside.md}" },
```

Comment:
Didn't we want to merge `padding-block-start` and `*-end` into one style rule? Or will we do that in a different PR?

Reply:
Yes, I’d prefer a separate change for that because it is a breaking change.